apollos-church-api/src/server.js

```diff
@@ -36,7 +36,7 @@ const isDev =
 
 const extensions = isDev ? [() => new RockLoggingExtension()] : [];
 
-const cacheOptions = isDev
+const cacheOptions = !isDev
```

Comment:
why the bang here? This will clear out cache options in production

Reply:
whoops - forgot to undo this.  Thanks for the code review!